### PR TITLE
feat: Adding PublicSectorBundle

### PR DESCRIPTION
**Pull request by sorbaugh**
* Resolves: # (none)

## Summary

This PR adds a new Bundle for public sectorr

## TODO

- [x] Extra approval step -> confirm if app list is final

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by jospoortvliet — APPROVED** [review]
LGTM!

File: lib/private/App/AppStore/Bundles/PublicSectorBundle.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getAppIdentifiers(): array {
+
+		return [
+			'files_confidential',
+			'forms',
+			'collectives',
+			'files_antivirus',
```
Comment (jospoortvliet) [inline lib/private/App/AppStore/Bundles/PublicSectorBundle.php:42]:
```suggestion
			'files_antivirus',
		    'twofactor_nextcloud_notification',
```

**Review by AndyScherzinger — APPROVED** [review]
Lgtm, how about also adding the audit app?

@sorbaugh @jospoortvliet

File: lib/private/App/AppStore/Bundles/PublicSectorBundle.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		    'twofactor_nextcloud_notification',
+			'tables',
+			'officeonline',
+			'logreader',
```
Comment (blizzz) [inline lib/private/App/AppStore/Bundles/PublicSectorBundle.php:46]: `logreader` this is already enabled by default. Can stay nevertheless of course.

**Review by blizzz — COMMENTED** [review]
:speech_balloon:

**Comment by blizzz** [comment]
> Lgtm, how about also adding the audit app?

loosely connected, but would be a reason to consider making https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html#log-level-interaction the default, or even enforced. It's a bit counter intuitive in general to switch this one through the log condition, and having audit_log enabled should be already sufficient for logging.

**Comment by AndyScherzinger** [comment]
Can ship with beta2 as is, so marketing can do their screenshots

**Comment by AndyScherzinger** [comment]
Fine to merge for Rc2 then 👍

File: lib/private/App/AppStore/Bundles/PublicSectorBundle.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ */
+namespace OC\App\AppStore\Bundles;
+
+class PublicSectorBundle extends Bundle {
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getName(): string {
+		return $this->l10n->t('Public bundle');
```
Comment (ChristophWurst) [inline lib/private/App/AppStore/Bundles/PublicSectorBundle.php:30]:
```suggestion
		return $this->l10n->t('Public sector bundle');
```

maybe

**Comment by sorbaugh** [comment]
Thanks!

**Pull request merged**